manage_app.py

```diff
@@ -130,28 +193,46 @@ def get_commit_url():
         print(bcolors.CYAN_IN + "BUILD HEALTH CHECK:" + bcolors.CEND)
 
         ###### WEB ######
-        get_web = check_service_up(WEB_URL, "WEB", False)
+        get_web = check_service_up(os.getenv("HUB_WEB_URL"), "WEB", False)
 
         # "connection reset by peer"
         print()
         time.sleep(10)
 
         ###### API ######
-        get_api = check_service_up(API_URL, "API", False)
+        get_api = check_service_up(os.getenv("HUB_API_URL"), "API", False)
+
+        ##### PY-API #####
+        get_py_api = check_service_up(
+            os.getenv("HUB_PY_API_URL"), "PY-API", False,
+        )
+
+        # URL-SHORTENER
+        get_url_shortener = check_service_up(
+            os.getenv("HUB_URL_SHORTENER"), "URL-SHORTENER", False,
+        )
 
         print()
         if (get_web == 200 and get_api == 400):
```

Comment:
Should check the status codes of the other services as well

Reply:
Done